appveyor.yml

```diff
@@ -1,5 +1,8 @@
 os: Visual Studio 2019
 
+init:
+ - netsh int ipv4 set dynamicport tcp start=1025 num=64510
```

Comment:
This one also requires Elevation:
"The requested operation requires elevation (Run as administrator)."

Will AppVeyor execute this as admin?

Reply:
This is OK, AppVeyor does not utilize UAC. AppVeyor log for this PR shows that the command is executed successfully.